_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -1101,6 +1103,18 @@ mfxStatus VAAPIEncoder::CreateAccelerationService(MfxVideoParam const & par)
         MFX_ERR_DEVICE_FAILED);
 
     // Configuration
+    std::vector<VAConfigAttrib> attrib_priority(1);
```

Comment:
I don't see the point of checking this attribute separately considering that we are checking other attributes down below. Can you, please, combine?
```
std::vector<VAConfigAttrib> attrib(2);
attrib[0].type = VAConfigAttribRTFormat;
attrib[1].type = VAConfigAttribRateControl;
attrib[2].type = VAConfigAttribContextPriority;
<...>
```

Reply:
hi Dvrogozh, the m_MaxContextPriority = attrib_priority[0].value only be read from media driver.   m_MaxContextPriority  don't need to be set by vaCreateConfig(). So I get the value independently.

Reply:
No, it is not. VAConfigAttribRTFormat is also only being read from the driver. I don't see the difference.

Reply:
I think I started to understand what you mean. do you mean that we pass these attributes to vaCreateConfig below and priority is not needed at this call?

If so, I would consider to move your query of the priority here:
https://github.com/Intel-Media-SDK/MediaSDK/blob/48f48caf166e5231cf2e2f60a97f0ecd9279a48b/_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp#L921

This is a common place where encoder gets driver capabilities.

Reply:
Yes, I only want to get the attribute of priority from driver. So I got it and move it to the query function.

Reply:
Done.